src/main/webapp/app/localci/build-queue/build-queue.service.ts

```diff
@@ -188,4 +189,16 @@ export class BuildQueueService {
             }),
         );
     }
+
+    /**
+     * Get all build jobs of a course in the queue
+     * @param buildJobId
+     */
+    getBuildJobLogs(buildJobId: string): Observable<BuildLogEntry[]> {
+        return this.http.get<BuildLogEntry[]>(`${this.resourceUrl}/build-log/${buildJobId}/entries`).pipe(
+            catchError((err) => {
+                return throwError(() => new Error(`Failed to get build log entries for build job ${buildJobId}\n${err.message}`));
```

Comment:
Is this error displayed in the UI? If this is the case we should use a translation instead